src/views/partials/sub-navigation/_macro.njk

```diff
@@ -1,20 +1,20 @@
 {% macro patternlibSubNavigation(params) %}
-  <nav class="patternlib-subnav" aria-label="{{ params.ariaLabel }}">
-      <ul class="patternlib-subnav__list" aria-label="{{ params.ariaListLabel }}">
+    <nav class="patternlib-subnav" aria-label="{{ params.ariaLabel }}">
+        <ul class="patternlib-subnav__list list list--bare" aria-label="{{ params.ariaListLabel }}">
         {% for item in (params.itemsList if params.itemsList is iterable else params.itemsList.items()) %}
-              <li class="patternlib-subnav__item {{ ' patternlib-subnav__item--active js-patternlib-subnav__item--active' if (item.url == params.currentPath) }}">
-                  <a href="{{ item.url }}" class="patternlib-subnav__link">{{ item.title }}</a>
-                  {% if item.anchors is defined and item.anchors %}
-                      <ul class="patternlib-subnav__sub-items">
-                          {% for anchor in item.anchors %}
-                              <li class="patternlib-subnav__item">
-                                  <a href="{{ anchor.url }}" class="patternlib-subnav__link">{{ anchor.title }}</a>
-                              </li>
-                          {% endfor %}
-                      </ul>
-                  {% endif %}
-              </li>
+                <li class="patternlib-subnav__item{{ ' patternlib-subnav__item--active' if (item.url == params.currentPath) }} list__item">
```

Comment:
`patternlib-subnav__item` and `patternlib-subnav__list` are no longer adding any css to the elements, so do we need them anymore?

Reply:
Should we keep that because we're still using `patternlib-subnav__item--active`?

Reply:
I see what you mean, but is there any point in having them if we never use them because were just using the css from lists anyway. They do make what the elements are more descriptive I suppose. Another thing we could do is something like this in the css:
```
patternlib-subnav__item {
  @extend .list__item;
}
```

Then we would only need the one class and it would still be descriptive